js/app.js

```diff
@@ -0,0 +1,517 @@
+  [
+    // These are Thom's shirts
+    "grey button-up",
+    "dark grey button-up",
+    "light blue button-up",
+    "blue button-up",
+  ],[
+    // These are Thom's pants
+    "grey jeans",
+    "jeans",
+    "PJs"
+  ],[
+    // Thom's accessories
+    "wool mittens",
+    "wool scarf",
+    "raybans"
+  ]
+];
+// What's Kristyn wearing today? Using bracket notation to access items in kristynsCloset, log the sentence "Kristyn is rocking that " + the third item in Kristyn's closet + " today!" to the console.
+// console.log("Kristyn is rocking that " + kristynsCloset[4] + " today!");
+// Kristyn just bought some sweet shades! Add "raybans" to her closet after "yellow knit hat".
+kristynsCloset.splice(6, 0, "raybans");
+// console.log(kristynsCloset);
+// Kristyn spilled coffee on her hat... modify this item to read "stained knit hat" instead of yellow.
+kristynsCloset[5] = "stained knit hat";
+// console.log(kristynsCloset);
+// Put together an outfit for Thom! Using bracket notation, access the first element in Thom's shirts array.
+thomsCloset[0] [0];
+// In the same way, access one item from Thom's pants array.
+thomsCloset[0];
+// Access one item from Thom's accessories array.
+thomsCloset[2];
+// Log a sentence about what Thom's wearing. Example: "Thom is looking fierce in a grey button-up, jeans and wool scarf!"
+console.log("Thom is looking fierce in a " + thomsCloset[0] [0] + " , " + "and " + thomsCloset[2] [1] + "!");
+// Get more specific about what kind of PJs Thom's wearing this winter. Modify the name of his PJ pants to Footie Pajamas
+thomsCloset[1][2] = "Footie Pajamas";
+console.log("Thom is wearing just " + thomsCloset[1][2] + " this winter.");
+//-----------------PART 4-----------------//
+//----------------printGreeting----------------//
+// Do you think you could write a function called printGreeting with a parameter name that returns a greeting with the argument interpolated into the greeting?
+//SKIPT
+//----------------printCool-------------------//
+// Write a function printCool that accepts one parameter, name as an argument. The function should print the name and a message saying that that person is cool.
+const printCool = (name) => {
+	console.log("Hey you, " + name + " you're pretty cool.");
+}
+printCool("Fil");
+//-------------------calculateCube-----------------//
+// Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+const calculateCube = (num) =>{
+	console.log(Math.pow(num, 3));
+}
+
+calculateCube(3);
+//-------------isVowel------------------//
+// Write a function isVowel that takes a character (i.e. a string of length 1) and 
+// returns true if it is a vowel, false otherwise. The vowel could be upper or lower case.
+
+const isVowel = (a) =>{
+const vowel = ["a", "e", "i", "o", "u"];
+for (let i = 0; i< vowel.length; i++){
+	if (a === vowel[i]) {
+		return true;
+	} else {
+		return false
+	}
+}
+}
+console.log(isVowel("f"));
```

Comment:
as written, this doesn't work.  it only returns true for 'a'.  see if you can repair it--lemme know if you want a hint.  also fix indentation.